public/lang/en.json

```diff
@@ -559,6 +559,7 @@
   "A5E.Settings.EnableConditionsHint": "Enable automation for the selected conditions.",
   "A5E.SettingsHideEmptyTraits": "Hide empty traits when the sheet is locked",
   "A5E.SettingsIncludeAbilityModifiersForSkills": "Include ability modifiers in skill modifiers",
+  "A5E.SettingsJackOfAllTrades": "Character has Jack of All Trades",
```

Comment:
If you look at the other settings here, they all specify what toggling this setting does, rather than saying why would should toggle it. In this case, I think it would also be acceptable to just call it "Jack of All Trades" and to provide a hint that describes what this will do. For example, "When enabled, half of your proficiency bonus (rounded down) will be added to all ability checks that do not already include your proficiency bonus."

Reply:
Actually, this should be skill or tool checks in A5e.